docs/userGuide/usingHtmlJavaScriptCss.md

```diff
@@ -61,5 +63,86 @@ Alternatively, you can use `<markdown>` (for _block_ Markdown elements such as h
 
 </div>
 
+## JavaScript libraries
+External JavaScript libraries can be included in MarkBind to add additional & wide-ranging features and functionalities. One such use case is to add a charting library for data visualisation.
+
+### Charts
```

Comment:
Think it would be better with a blank line after the heading to [follow code style!](https://se-education.org/guides/conventions/markdown.html)